apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -169,9 +168,14 @@ type Application struct {
 // +docgen:ignore
 type Infrastructure struct{}
 
-// Audit enables audit logs. Filtering may be added in future.
-// +docgen:ignore
-type Audit struct{}
+type Audit struct {
+	// APIPolicyRef is the name of an APIAuditLogPolicy resource in the same name-space.
+	APIPolicyRef string `json:"apiPolicyRef,omitempty"`
+
+	// EnableNode enables operating system audit logs from nodes.
+	// If not set, node audit logs are not forwarded.
+	EnableNode bool `json:"enableNode,omitempty"`
```

Comment:
- Does this need to default to "true" to support upgrade?  
- This says "OS audit logs" which are... linuxd logs?  is there an implication here? How do kube logs fit into this?
- Can there be more then one like user defined app logs?  Should this have a name too?

Reply:
Removed in latest version. Policy is a "filter", no changes to inputs. Policy filter can operate on any stream of logs, will only act on records that look like API audit events.